hystrix-contrib/hystrix-javanica/hystrix-javanica-aspectj/src/main/java/com/netflix/hystrix/contrib/javanica/aop/aspectj/MethodExecutionAction.java

```diff
@@ -39,17 +41,17 @@
     private final Object object;
     private final Method method;
     private final Object[] _args;
-    private final MetaHolder metaHolder;
+    private final AspectjMetaHolder metaHolder;
```

Comment:
I think we cannot keep `MethodExecutionAction` in core because all `execute` methods are different for aspectj and aop-alliance.